tests/integration-tests/configs/common/common.yaml

```diff
@@ -168,12 +168,12 @@ disable_hyperthreading:
       # HT disabled via CpuOptions
       - regions: ["us-west-1"]
         instances: ["c5.xlarge"]
-        oss: ["ubuntu1804"]
+        oss: ["ubuntu1804", "ubuntu2004"]
         schedulers: ["slurm"]
   test_disable_hyperthreading.py::test_sit_disable_hyperthreading:
     dimensions:
       # Manually disabled HT
-      {%- for os, scheduler in [("centos7", "torque"), ("ubuntu1804", "sge")] %}
+      {%- for os, scheduler in [("ubuntu2004", "torque"), ("ubuntu1804", "sge")] %}
```

Comment:
Is this change from centos7 to ubuntu20 intended?

Reply:
Yes, the reasoning behind being that centos7 with manually disabled ht is already tested in `test_hit_disable_hyperthreading`. So this way both `centos7` and `ubuntu20` will be tested in both use cases. What do you think?